### DISCUSSION TITLE
Protocol analyzers not showing up

### DISCUSSION THREAD

**Discussion by Dmartynowych** [gusmanb/logicanalyzer#62] [Q&A] [answered]
Almost certainly a function of me not knowing what I'm am doing while build this but I'm not seeing an of the protocol analyzers showing up on either windows or mac os - otherwise the logic analyzer appears functional

**Comment by gusmanb** [comment]
Have you copied the analyzer DLL's to the correct folder?

https://github.com/gusmanb/logicanalyzer/wiki/06---The-LogicAnalyzer-program#protocol-analyzers-menu

**Reply by Dmartynowych** [reply]
i don't see an "analyzers" directory (what would the relative path be?) - i tried adding the DLL to the directory containing the executable but figured that wasn't correct

**Reply by gusmanb** [reply, marked as answer]
You must create it as you are compiling from sources, in the compiled packages the folder and libraries are in place. The directory must be in the folder of the application.

**Reply by Dmartynowych** [reply]
perfect! thank you - this is a great tool!!
